geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedAckRegionCCEDUnitTest.java

```diff
@@ -243,7 +240,8 @@ public void run() {
               ClusterDistributionManager.NORMAL_DM_TYPE, null, null);
           tag.setMemberID(mbr);
         } catch (UnknownHostException e) {
-          org.apache.geode.test.dunit.Assert.fail("could not create member id", e);
+          e.printStackTrace();
```

Comment:
I think calling "printStackTrace" is usually a bad idea. I have had experiences when I could not track down the stack trace for the fail message.
The deprecation on our dunit.Assert says to use Assertions instead.
In AssertJ's Assertions I see it has this:
  public static void fail(String failureMessage, Throwable realCause) {
    Fail.fail(failureMessage, realCause);
  }
I think these places you found  that called our old fail(String, Throwable) could just be converted to call this assertj fail(String, Throwable) without needing to add the printStackTrace call